server.js

```diff
@@ -4439,6 +4450,10 @@ function objectWebServer() {
                     var tempFolderName2 = utilities.readObject(objectLookup, req.body.name);//req.body.name + thisMacAddress;
                     // remove object from tree
                     if (tempFolderName2 !== null) {
+                        if (activeHeartbeats[tempFolderName2]) {
+                            clearInterval(activeHeartbeats[tempFolderName2]);
+                            delete activeHeartbeats[tempFolderName2];
+                        }
```

Comment:
This is good. I guess in the past, heartbeats of deleted objects continued to be broadcast until we restarted the server, but your new code is the correct behavior.